File: examples/indexer/javascript/accountsApplicationID.js
// AccountsAssetID.js
// requires algosdk@1.6.1 or higher 
// verify installed version
// npm list algosdk

const algosdk = require('algosdk');
const chalk = require('chalk');
// const indexer_token = "";
// const indexer_server = "http://localhost";
// const indexer_port = 8980;

// const indexer_server = "https://testnet-algorand.api.purestake.io/idx2/";
// const indexer_port = "";
// const indexer_token = {
//     'X-API-key': 'B3SU4KcVKi94Jap2VXkK83xx38bsv95K5UZm2lab',
// }
const indexer_server = "http://localhost";
const indexer_port = 59998;
const indexer_token = "";

// const indexer_server = "https://indexer-internal-testnet.aws.algodev.network";
// const indexer_port = 443;
// const indexer_token = "YddOUGbAjHLr1uPZtZwHOvMDmXvR1Zvw1f3Roj2PT1ufenXbNyIxIz0IeznrLbDsF"

 
// Instantiate the indexer client wrapper
let indexerClient = new algosdk.Indexer(indexer_token, indexer_server, indexer_port);

(async () => {
    let applicationID = 70;  
    let accountInfo = await indexerClient.searchAccounts()
        .applicationID(applicationID).do();
    console.log(chalk.black("Information for account info for Application ID: " + JSON.stringify(accountInfo, undefined, 2)));
})().catch(e => {
    console.log(e);
    console.trace();
});

File: examples/indexer/python/accounts_application_id.py
# accounts_assetid.py
import json
# requires Python SDK version 1.3 or higher
from algosdk.v2client import indexer

# instantiate indexer client

# myindexer_token = 'B3SU4KcVKi94Jap2VXkK83xx38bsv95K5UZm2lab'
# myindexer_address = 'https://betanet-algorand.api.purestake.io/idx2/'
# myindexer_header = {'X-Api-key': myindexer_token}

myindexer_token = 'B3SU4KcVKi94Jap2VXkK83xx38bsv95K5UZm2lab'
myindexer_address = 'https://betanet-algorand.api.purestake.io/idx2/'
myindexer_header = {'X-Api-key': myindexer_token}

# myindexer_token = 'YddOUGbAjHLr1uPZtZwHOvMDmXvR1Zvw1f3Roj2PT1ufenXbNyIxIz0IeznrLbDsF'
# myindexer_address = 'https://indexer-internal-betanet.aws.algodev.network:443'
# myindexer_header = {'X-Indexer-API-Token': myindexer_token}

# https://indexer-internal-beta//net.aws.algodev.network

# myindexer = indexer.IndexerClient(indexer_token="", indexer_address="http://localhost:8980")
# myindexer = indexer.IndexerClient(
#     indexer_token="", indexer_address=myindexer_address, headers=myindexer_header)
myindexer = indexer.IndexerClient(
    indexer_token="", indexer_address=myindexer_address, headers=myindexer_header)

response = myindexer.accounts(
    application_id=2672020)
print("Account Info: " + json.dumps(response, indent=2, sort_keys=True))

